releases/release-1.11/release-1.11.md

```diff
@@ -0,0 +1,126 @@
+# Kubernetes 1.11 Release Schedule
+
+*Handy Links (not populated yet):*
+
+* [Release Team](http://bit.ly/kube111-team)
```

Comment:
nit: last 2 releases used http://bit.ly/k8sXX-type convention.  The kubeXXretro is legacy going back to 1.3 and is the only link that breaks the convention.

Reply:
ah, I wasn't aware there was a tradition.  the first time I saw the links was 1.10.